StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/PaymentMethodRowButton.swift

```diff
@@ -25,16 +27,42 @@ final class PaymentMethodRowButton: UIView {
         // TODO(porter) Add can remove and can update
     }
 
+    enum State {
+        case selected
+        case unselected
+        case editing
+    }
+
     // MARK: Internal properties
-    // TODO(porter) Maybe expand this into an enum of (selected, unselected, editing) state
+    var state: State = .unselected {
+        didSet {
+            previousState = oldValue
+
+            selectionTapGesture.isEnabled = !isEditing
+            shadowRoundedRect.isSelected = isSelected
+            circleView.alpha = isSelected ? 1.0 : 0.0
```

Comment:
For some reason `isHidden` doesn't work on the `circleView`, not sure if it's something with the drawing code or something.